Lesson4.1/src/Computer.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Random;
+
+public class Computer {
+    private final String CPU = "AMD 4GHz 8 core";
+    private final String SSD = "512GB";
+    private final String RAM = "8GB";
+    private int countLifeCycle = 10;
+    private int randomNumber;
+    private ComputerCondition computerCondition;
+    Random random = new Random();
+
+    public Computer() {
+        randomNumber = random.nextInt(2);
```

Comment:
you don't need to store it. call random.nextInt(2) when you need to get a  random number